mandatory/3-function-output.js

```diff
@@ -1,18 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
-  return Math.random() * 10;
+  return Math.random() * 10;  // it returns a random number that can be anywhere between 0 and 1. The 0 is included and 1 is excluded.
```

Comment:
Comment is clear, well done. 
You've described the function [Math.random ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) well, but note the **multiply by 10**. What do you think the function returns with the **multiply by 10**?